src/main/java/tracko/logic/commands/item/EditItemCommand.java

```diff
@@ -45,7 +45,8 @@ public class EditItemCommand extends Command {
             + CliSyntax.PREFIX_TAG + "Limited "
             + CliSyntax.PREFIX_TAG + "New";
 
-    public static final String MESSAGE_EDIT_ITEM_SUCCESS = "Edited Item: \n%1$s";
+    public static final String MESSAGE_EDIT_ITEM_SUCCESS = "Edited Item:\n%1$s";
```

Comment:
Should there be a space here? Minor nitpick, shouldn't affect merge and can fix in next PR if needed. 
```suggestion
    public static final String MESSAGE_EDIT_ITEM_SUCCESS = "Edited Item: \n%1$s";
```

Reply:
Thanks Janelle! UI-wise, it doesn't really affect the display, but if there is a space there, it will introduce trailing whitespaces, such as:
Edited item:` `
information here

instead of

Edited item:
information here

I personally think the version without the trailing whitespaces is better, but let me know what you think as well!